src/Storage/Device/Ceph.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Utopia\Storage\Device;
+
+class Ceph extends S3
+{
+    /**
+     * Regions constants
+     */
+    const US_EAST_1 = 'us-east-1';
+
+    /**
+     * Ceph Constructor
+     *
+     * @param  string  $root
+     * @param  string  $accessKey
+     * @param  string  $secretKey
+     * @param  string  $bucket
+     * @param  string  $region
+     * @param  string  $acl
+     */
+    public function __construct(string $root, string $accessKey, string $secretKey, string $bucket, string $region = self::US_WEST_001, string $acl = self::ACL_PRIVATE)
+    {
+        parent::__construct($root, $accessKey, $secretKey, $bucket, $region, $acl);
+        $this->headers['host'] = $bucket.'.'.'s3'.'.'.$region.'.ceph.io';
```

Comment:
Do you have any resources/docs for what the host should be?

Reply:
Hey, as per the use I guess I can modify it to 
` $this->headers['host'] = $cephHost;`
because Ceph, being a distributed storage system, does not have a single centralized API endpoint like some cloud storage providers (e.g., Amazon S3 or Backblaze B2). Instead, Ceph provides APIs through its various components, and the endpoint hostname varies depending on the specific Ceph service you want to interact with.  So it would me more meaningful to get that parameter from user? 
What's your thoughts on this?